xunit.runner.wpf/MainWindow.xaml

```diff
@@ -98,7 +98,29 @@
                     </ListBox>
                     <Label Content="Traits:"
                            Grid.Row="4" />
-                    <ListBox Grid.Row="5" />
+                    <ListBox Grid.Row="5"
+                             ItemsSource="{Binding Traits}">
+                        <ListBox.ItemTemplate>
+                            <DataTemplate DataType="vm:TraitViewModel">
+                                <TextBlock Text="{Binding DisplayName}" />
+                            </DataTemplate>
+                        </ListBox.ItemTemplate>
+                        <ListBox.ItemContainerStyle>
+                            <Style TargetType="{x:Type ListBoxItem}">
+                                <Setter Property="IsSelected" Value="{Binding Mode=TwoWay, Path=IsSelected}"/>
+                            </Style>
+                        </ListBox.ItemContainerStyle>
+                        <ListBox.ContextMenu>
+                            <ContextMenu>
+                                <MenuItem Header="Clear" Command="{Binding TraitsClearCommand}" />
```

Comment:
The old xunit gui also supported Escape to clear at least some filters.  We should consider that as well as a clunky right-click menu.